[2025-02-08T19:00:45.987+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T19:00:46.002+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-12-11T00:00:00+00:00 [queued]>
[2025-02-08T19:00:46.009+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-12-11T00:00:00+00:00 [queued]>
[2025-02-08T19:00:46.009+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T19:00:46.019+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_account_dim_data> on 2024-12-11 00:00:00+00:00
[2025-02-08T19:00:46.027+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=855) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T19:00:46.028+0000] {standard_task_runner.py:72} INFO - Started process 871 to run task
[2025-02-08T19:00:46.027+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_dim_gen', 'generate_account_dim_data', 'scheduled__2024-12-11T00:00:00+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/account_dim_gen.py', '--cfg-path', '/tmp/tmpzf_g08u9']
[2025-02-08T19:00:46.029+0000] {standard_task_runner.py:105} INFO - Job 96: Subtask generate_account_dim_data
[2025-02-08T19:00:46.067+0000] {task_command.py:467} INFO - Running <TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-12-11T00:00:00+00:00 [running]> on host 2c1cd59afcc0
[2025-02-08T19:00:46.129+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='account_dim_gen' AIRFLOW_CTX_TASK_ID='generate_account_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-11T00:00:00+00:00'
[2025-02-08T19:00:46.130+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T19:00:46.144+0000] {logging_mixin.py:190} INFO - CSV file ./account_dim_large_data.csv with 50 row has been generated successfully!
[2025-02-08T19:00:46.144+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T19:00:46.151+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T19:00:46.151+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=account_dim_gen, task_id=generate_account_dim_data, run_id=scheduled__2024-12-11T00:00:00+00:00, execution_date=20241211T000000, start_date=20250208T190046, end_date=20250208T190046
[2025-02-08T19:00:46.202+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T19:00:46.215+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T19:00:46.217+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:40:49.921+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:40:49.934+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-12-11T00:00:00+00:00 [queued]>
[2025-02-08T20:40:49.943+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-12-11T00:00:00+00:00 [queued]>
[2025-02-08T20:40:49.943+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:40:49.956+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_account_dim_data> on 2024-12-11 00:00:00+00:00
[2025-02-08T20:40:49.965+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1696) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:40:49.967+0000] {standard_task_runner.py:72} INFO - Started process 1714 to run task
[2025-02-08T20:40:49.966+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_dim_gen', 'generate_account_dim_data', 'scheduled__2024-12-11T00:00:00+00:00', '--job-id', '267', '--raw', '--subdir', 'DAGS_FOLDER/account_dim_gen.py', '--cfg-path', '/tmp/tmpg3gzs20c']
[2025-02-08T20:40:49.968+0000] {standard_task_runner.py:105} INFO - Job 267: Subtask generate_account_dim_data
[2025-02-08T20:40:50.014+0000] {task_command.py:467} INFO - Running <TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-12-11T00:00:00+00:00 [running]> on host 7e959c1a2a8b
[2025-02-08T20:40:50.084+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='account_dim_gen' AIRFLOW_CTX_TASK_ID='generate_account_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-11T00:00:00+00:00'
[2025-02-08T20:40:50.085+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:40:50.102+0000] {logging_mixin.py:190} INFO - CSV file ./account_dim_large_data.csv with 50 row has been generated successfully!
[2025-02-08T20:40:50.102+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:40:50.109+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:40:50.109+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=account_dim_gen, task_id=generate_account_dim_data, run_id=scheduled__2024-12-11T00:00:00+00:00, execution_date=20241211T000000, start_date=20250208T204049, end_date=20250208T204050
[2025-02-08T20:40:50.145+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:40:50.166+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:40:50.169+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
